common-npm-packages/codecoverage-tools/utilities.ts

```diff
@@ -111,7 +113,9 @@ export function insertTextToFileSync(filePath: string, prependFileContent?: stri
             let appendBuffer = new Buffer(appendFileContent);
             fs.writeSync(fd, appendBuffer, 0, appendBuffer.length, existingData.length + preTextLength);
         }
-        fs.close(fd);
+        fs.close(fd, (err) => {
+            tl.warning(err.message);
```

Comment:
Should there be check for err? Since err could be empty for successful completion

Reply:
@anatolybolshakov Thank you! I've deleted warnings. Could you please take another look?